oeps/processes/oep-0056-architectural-advisory-process.rst

```diff
@@ -0,0 +1,197 @@
+
+We will necessarily iterate on the organization of the checklist as we learn more about how we use it.  This OEP also does not prescribe where the checklist needs to be written down.
+
+Other Options Considered
+========================
+
+Synchronous Architecture Advisory Forum meeting
+-----------------------------------------------
+
+The approach to the Advisory Process advocated for in `Scaling the Practice of Architecture, Conversationally <conversationally_>`_ centers around the "Architecture Advisory Forum", a synchronous meeting where the group reviews ADRs, Architectural Principles, Tech Radar blips, etc.
+
+We see three significant problems with the idea of having such a forum:
+
+- Our community is global.  Any time we could propose for this meeting would necessarily leave out a significant portion of our stakeholders and experts.
+
+- Our community is multi-faceted.  ADRs may have completely divergent sets of stakeholders.  How do we manage the guest list of such a meeting to ensure that the right voices are speaking on any given ADR, and that the right individuals are in the room?  It's a logistical nightmare given the size and scope of our community.
+
+- Our community is comprised of many independent organizations.  There are times when our component organizations may want to use the Advisory Process on "internal" decisions as well.  Do we create a separate forum meeting for those decisions?  How do we then ensure that we err on the side of having an open, public forum?  Again, logistically, this is quite difficult.
+
+.. _principles:
+
+Appendix A: Advisory Process Principles
+=======================================
+
+The creation of the Advisory Process is a fundamental shift in expectations of how we conduct our architectural process, and that shift requires an investment of time in order to be successful. The principles below describe the expectations, spirit and benefits of this process.
+
+Asynchronous First
+------------------
+
+Our community is global, and so we will prioritize asynchronous communication in the Architecture Advisory Process. This means it's important that individuals be able to participate in the process even if they cannot attend a synchronous meeting.  We believe there's value in getting together synchronously, but the process should prioritize getting the right input over having synchronous meetings.  That said, if the relevant stakeholders and experts are able to schedule an effective in person meeting, there's additional value in getting together synchronously when possible.  Note, however, that the decision makers *still have the responsibility of recording in their ADR the advice they received.*
```

Comment:
1. Once you determine your stance on whether there will or will not be any "synchronous" parts to this process, this section should be cleaned-up/clarified.
2. [food for thought] Related, I wanted to note some additional differences between sync and async that we may want to acknowledge and/or experiment around:
  * See [Architecture Advisory Forum section of article](https://martinfowler.com/articles/scaling-architecture-conversationally.html#forum) for details. 
  * A threaded discussion is not the same as a conversation. It is better in many ways, but may not be as good in others. Do we want allow for conversations as well that get documented on the PR?
  * The meeting would have had a variety of attendees that would see all ADRs. Although not made explicit, I imagine that this provides an implicit check-and-balance point for the process. Maybe it would better ensure that the right stakeholders were asked, or that we come back to the Architecture Principles or the Tech Radar, or a place to iterate on the process.  I'm not saying there must be a synchronous meeting, but we may want additional experimentation around replacing this, rather that just skipping it, which is where I think we've landed. Possible alternatives:
    * Include a standard set of stakeholders on all ADRs, in addition to any specific stakeholders. (I don't like this.)
    * Set up an (estimated) 1 hour meeting that a set of individuals could asynchronously attend, but might have some meeting notes around what ADRs to review, etc. (See other topics as well.)
  * Other parts of the agenda noted were "sharing spikes / upcoming decisions" and "re-visiting decisions", and others like "reviewing collective key metrics" (which I think the author didn't really go into). Are these things we will need to replace in some other way?

Basically, it feels like we're planning on experimenting with someone else's process, but we are dropping a big chunk of what they labelled: "A [regular] time and place for conversations: The Architecture Advisory Forum".

Reply:
I started the OEP with the idea of a synchronous meeting, like the article suggests, and was daunted by the logistics of making it useful and productive for our community.  (That thought process is covered in the "Other Options Considered" section so I won't write it again here.)  

After reading the article and thinking about the Open edX Community, I just didn't see how we could get away with the process as-written and so tried to reframe it in an async, working-in-public-style way, which I think we can all agree is a principle we want to promote in our community anyway.

That said, in the "Asynchronous First" section above, I wanted to _at least_ acknowledge the value of synchronous time while framing it in the context of the larger asynchronous process.  Likewise, in the "Architectural Advisory Process (AAP)" introduction, the OEP also takes a stab at how we can use the Process - even without a meeting - to get some of the same benefits of the Architectural Advisory Forum but in an asynchronous way.  

> We intend to also use the Advisory Process over time to revisit and iterate on our architectural principles with the goal of ensuring they're specific, measurable, achievable, realistic, and testable (a variation of SMART criteria).  Similarly, like ADRs and architectural principles, we can use the process to review and adjust Open edX Tech Radar blips.

Admittedly this isn't the focus of the OEP, but it felt worth getting in there, and could benefit from some of the same post-acceptance nuts-and-bolts iteration that we intend for everything else.

I'm not sure how I can clean up/clarify this section, as per your first point... do you have any suggestions?  To me it seems clear that the process is asynchronous, but that doesn't preclude synchronous meetings where possible/reasonable.  Personally I was happy with it as-is, but certainly welcome input.